packer/windows/conf/bin/bk-install-elastic-stack.ps1

```diff
@@ -227,6 +228,9 @@ If ((![string]::IsNullOrEmpty($Env:BUILDKITE_ENV_FILE_URL)) -And (Test-Path -Pat
   }
 }
 
+nssm set buildkite-agent AppEnvironmentExtra BUILDKITE_TERMINATE_INSTANCE_AFTER_JOB=$Env:BUILDKITE_TERMINATE_INSTANCE_AFTER_JOB
+If ($lastexitcode -ne 0) { Exit $lastexitcode }
```

Comment:
This line repeats the next one - I'm guessing that the existing one was checking the exit code from the `Get-Content` in the loop above. So maybe the lines should be swapped.